### ISSUE TITLE
Multiple "pagelist_selected" on a page always show the same (first) selection (pagelist 3.8)

### ISSUE THREAD

**Issue by idmedia-fritscher** [t3brightside/pagelist#24]
I am experiencing a problem with pagelist 3.8: If I put multiple items of CType "pagelist_selected" on a page, with different selected pages, the items on the page will always show up with the pagelist selection of the first item.
I debugged the data going into `Pagelist.html` and it shows `data.pages` as a string with the correct values (comma separated list of pages), different for each item. But `pagelist` is always an array of the same entries.
I just recently updated pagelist from an old version and I am myself quite new to Typo3 so there is chance I misconfigured something.
Can anyone help me debug this?
Thanks

**Comment by t3brightside** [comment]
Indeed, well spotted @idmedia-fritscher, a bug it is, will look into it, hopefully soon!
